extensions/source-support/src/test/java/io/deephaven/generic/region/AppendOnlyFixedSizePageRegionTest.java

```diff
@@ -159,6 +162,22 @@ public void run() {
                 }
             });
         }
+
+        @Override
+        public LogOutput append(LogOutput logOutput) {
+            return logOutput.append("DependentRegistrar[")
+                    .append(LogOutput.APPENDABLE_ARRAY_FORMATTER, dependencies).append(']');
+        }
+
+        @Override
+        public boolean satisfied(final long step) {
+            return updateGraph.satisfied(step);
```

Comment:
I don't understand this.

My basic concern with it is: (1) we’re using the updateGraph to say we are satisfied or not.  We enqueue a notification in run() [and satisfied could occur any time after the run].  That notification may or may not be able to execute, based on those N dependencies.

But if you have the DependentRegistrar as a parent, it would have fired and have enqueued a notification that will fire sometime in the future, depending on those dependencies.  But you would never ask about them, only about the satisfaction of the Dependent registrar.

In the simple blink table case, I think I get it.  Because the update graph is only satisfied after the blink table calls it’s run() method, which will have created the notifications for the things localTable depends on.  In the dependent case, I’m just not certain why this is correct.